recipes-kernel/linux-firmware/linux-firmware_git.bbappend

```diff
@@ -0,0 +1,25 @@
+PR .= ".1"
```

Comment:
This is fine, but shouldn't be required any more because of the jethro uprev (with the prserver).